pandas/tests/series/test_apply.py

```diff
@@ -576,3 +576,13 @@ def f(x):
         result = s.map(f)
         exp = pd.Series(['Asia/Tokyo'] * 25, name='XX')
         tm.assert_series_equal(result, exp)
+
+    @pytest.mark.parametrize("vals,mapping,exp", [
+        (list('abc'), {np.nan: 'not NaN'}, [np.nan] * 3 + ['not NaN']),
+        (list('abc'), {'a': 'a letter'}, ['a letter'] + [np.nan] * 3),
+        (list(range(3)), {0: 42}, [42] + [np.nan] * 3)])
+    def test_map_missing_mixed(self, vals, mapping, exp):
+        s = pd.Series(vals + [np.nan])
```

Comment:
Can you add `# GH20495` as a comment on its own line right below the method definition?

Reply:
Between `def ...` and `s ...`?

Reply:
Yes - you can look at `test_with_nested_series` in the same module for reference